### PR TITLE
Feature/desktop-version

### PR DESCRIPTION
**Pull request by sazukeR**
Hello partner, this is the desktop-version of my project, i'll be greatfull for your suggestion wheen you have a chance. I know that you must very busy 👍 
dv

### CODE REVIEW

**Review by Edjhernandez — CHANGES_REQUESTED** [review]
One last comment, in the mobile version i had seen that your burger-button changes color to black when the background turned white, now it is white and cannot be seen.
Could you explain to me the reason for this behavior?

Good job, go ahead!

File: form.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                            <input type="text" placeholder="Write your name..." name="name" id="name" pattern="^[A-Za-zÑñÁáÉéÍíÓóÚúÜü\s]+$" title="⚠ Name only acepts letters and white spaces" required>
+                        </div>
+                        <div>
+                            <label for="email">Email</label>
+                            <input type="email" placeholder="Write your email..." name="email" id="email" pattern="^[a-z0-9]+(\.[_a-z0-9]+)*@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,15})$" title="⚠ Wrong email" required>
+                        </div>
+                        <div>
+                            <label for="message">Message</label>
+                            <textarea cols="15" rows="5" placeholder="Write a message..." data-pattern="^.{1,255}$" title="⚠ Your comment must not exceed 255 characters" name="message" id="message" required></textarea>
+                        </div>
```
[the remark this answers is not in the document]
Comment (sazukeR) [reply form.html:67]: I have thought about deploying these validations more carefully with the programming

File: about-me.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    </head>
+    <body class="pages-body">
+     <header>
+        <div class="navi">
+            <a class="logo" href="#">
+                <figure>
+                    <img src="assets/bw.png" alt="sasukeR">
+                    <figcaption>sasukeR</figcaption>
+                </figure>
+            </a>
```
[the remark this answers is not in the document]
Comment (sazukeR) [reply about-me.html:27]: this is the way to use this buttons, are used to redirect at the begin or section home of the website

File: about-me.html [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+                    </li>
+                    <li>
+                        <h6><i class="fa-brands fa-twitter"></i> Twitter</h6>
+                        <span>https://twitter.com/reinaldojcg1</span>
+                    </li>
+                    <li>
+                        <h6><i class="fa-brands fa-github"></i> GitHub</h6>
+                        <span>https://github.com/sazukeR/sazukeR</span>
+                    </li>
+                    <li>
```
[the remark this answers is not in the document]
Comment (sazukeR) [reply about-me.html:88]: ok 👍

**Pull request merged**